### PR TITLE
[feat] Adding RenderPixelatedPass types. 

### LINKED ISSUE #301: Add RenderPixelatedPass
### Describe the feature you'd like:
As per [r147 migration guide](https://github.com/mrdoob/three.js/wiki/Migration-Guide#146--147), `PixelShader` was replaced with `RenderPixelatedPass` inside the `jsm/examples` folder. I think we should add types for this new pass. 😄 

### Suggested implementation:
Adding the class types should be helpful for those who want to use this composer pass.
I can make a quick PR to fix this for the [r147](https://github.com/three-types/three-ts-types/issues/276) release.

### PR DESCRIPTION
**Pull request by LuchoTurtle**
I know files in the `examples` folder is not a priority but I felt this would be useful for anyone wanting to make use of this pass. :]

### Why
This should close #301 .
It follows the specifications defined in the [r147 migration guide](https://github.com/mrdoob/three.js/wiki/Migration-Guide#146--147).

> `PixelShader` has been removed. Use `postprocessing/RenderPixelatedPass` instead.

### What
Added the types for `RenderPixelatedPass` inside `jsm/examples/postprocessing/RenderPixelatedPass.d.ts`.
Also removed `PixelShader` inside the `shaders` folder.

### Checklist

-   [x] Checked the target branch (current goes `master`, next goes `dev`)
-   [x] Added myself to contributors table
-   [x] Ready to be merged

### CODE REVIEW

**Comment by LuchoTurtle** [comment]
CI failed because I mistyped the type declaration file in `OTHER_FILES.txt`. Should be fixed now 😅

**Comment by 0b5vr** [comment]
three.js side: https://github.com/mrdoob/three.js/blob/r147/examples/jsm/postprocessing/RenderPixelatedPass.js

File: types/three/examples/jsm/postprocessing/RenderPixelatedPass.d.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+export class RenderPixelatedPass extends Pass {
+    constructor(pixelSize: number, scene: Scene, camera: Camera, options?: RenderPixelatedPassParameters);
+    pixelSize: number;
+    resolution: Vector2;
+    renderResolution: Vector2;
+
+    pixelatedMaterial: ShaderMaterial;
+    normalMaterial: MeshNormalMaterial;
+
+    fsQuad: object;
```
Comment (0b5vr) [inline types/three/examples/jsm/postprocessing/RenderPixelatedPass.d.ts]: this is `FullScreenQuad` . It should be accessible from `./Pass`

https://github.com/three-types/three-ts-types/blob/r146/types/three/examples/jsm/postprocessing/Pass.d.ts

**Review by 0b5vr — CHANGES_REQUESTED** [review]
I think it's already on the mergeable degree but if you'd change the `FullScreenQuad` part it would be nice.

File: types/three/examples/jsm/postprocessing/RenderPixelatedPass.d.ts (hunk 1 above)
Comment (LuchoTurtle) [reply types/three/examples/jsm/postprocessing/RenderPixelatedPass.d.ts]: Should be fixed on [`05af4d0` (#302)](https://github.com/three-types/three-ts-types/pull/302/commits/05af4d022cd093c29d746f91dfa823a376afc96b).
Sorry about that!

**Comment by LuchoTurtle** [comment]
Done 👍 
I've noticed the `fsQuad` field in various other files are defined as `fsQuad: object`. 
Would it make sense to open a different PR to change to `FullScreenQuad`?

**Review by 0b5vr — APPROVED** [review]

**Comment by 0b5vr** [comment]
Thank youuu!!

> I've noticed the `fsQuad` field in various other files are defined as `fsQuad: object`. Would it make sense to open a different PR to change to `FullScreenQuad`?

I believe that will be very helpful, yes!

**Comment by LuchoTurtle** [comment]
Great!
I'll see if I have time to do that! 😄

**Comment by LuchoTurtle** [comment]
> Thank youuu!!
> 
> > I've noticed the `fsQuad` field in various other files are defined as `fsQuad: object`. Would it make sense to open a different PR to change to `FullScreenQuad`?
> 
> I believe that will be very helpful, yes!

Got a PR https://github.com/three-types/three-ts-types/pull/305 up regarding this.
Let me know if there's anything I should fix or not :]

**Pull request merged**